### PR TITLE
Update PhantomJS to 2.1.1, remove 2.0 warning in readme

### PR DESCRIPTION
**Pull request by mikesherov**

### CODE REVIEW

**Comment by skilledDeveloper** [comment]
+1

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "phantomjs",
-  "version": "1.9.19",
+  "version": "2.1.0",
```
Comment (nicks) [inline package.json]: ya....i'm on the fence about whether it should be `2.1.0` vs `2.1.1`...technically this is more correct, but we're going to get a bunch of issues filed for "update to 2.1.1"

**Comment by nicks** [comment]
lgtm, i'm just going to merge and reconcile. thanks!

**Comment by booleanbetrayal** [comment]
woooo!

**Comment by jzaefferer** [comment]
Nice. Just need a tag and publish to npm now.

**Pull request merged**